test/ext/test_horizontal_shard.py

```diff
@@ -226,6 +228,21 @@ def test_get(self):
         t2 = sess.get(WeatherLocation, 1)
         is_(t2, tokyo)
```

Comment:
**mike bayer** (zzzeek) wrote:

Done

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4872